app/scripts/controllers/home.ctrl.js

```diff
@@ -3,13 +3,16 @@
 angular.module('yourStlCourts').controller('HomeCtrl', function (Citations,toaster,States,municipalities,$uibModal,$state) {
   var ctrl = this;
   ctrl.states = States;
+  //modifyMunicipalities should be called and used before ctrl.municipalities is initialized
+  //because it will also modify the original municipalties object to remove the "Unincorporated" Key Word
+  ctrl.modifiedMunicipalities = modifyMunicipalities(municipalities);
   ctrl.municipalities = municipalities;
   ctrl.selectedMunicipality = null;
   ctrl.citationCriteria = {};
   ctrl.hasEverSelected = false;
 
   ctrl.municipalitySelected  = function(){
-    $state.go('courtSearchInfo',{courtId : ctrl.selectedMunicipality.court_id});
+    $state.go('courtSearchInfo',{'courtId' : ctrl.selectedMunicipality.court_id});
```

Comment:
This change doesn't do anything. Typically, in JS, keys don't have string tics around them.